persistence-jdbc/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/slick/SlickReadSide.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.scaladsl.persistence.slick
+
+import slick.dbio.{ DBIOAction, NoStream }
+import slick.jdbc.JdbcProfile
+import slick.jdbc.JdbcBackend.Database
+import com.lightbend.lagom.scaladsl.persistence._
+
+import scala.concurrent.ExecutionContext
+import scala.reflect.ClassTag
+
+trait SlickReadSide {
```

Comment:
I think we'll need to add API docs to this trait, since it will be part of the public API.